server/libmain/libmain.go

```diff
@@ -342,6 +342,12 @@ func registerLocalGRPCClients(env *real_environment.RealEnv) error {
 	if env.GetActionCacheServer() != nil {
 		env.SetActionCacheClient(repb.NewActionCacheClient(conn))
 	}
+	if env.GetBuildBuddyServer() != nil {
+		env.SetBuildBuddyServiceClient(bbspb.NewBuildBuddyServiceClient(conn))
```

Comment:
i don't think you'll actually want to register bbservce as a grpc service on a proxy machine: it'll probably break rpc forwarding for the services that you don't implement, and the only solution i can think of for that (creating a `buildbuddy_server` without actually registering it) feels kinda brittle since it'll still be unclear which services even work.

maybe you'd be better off just throwing the Metadata call into a bb-only rpc that we define on the CAS service and throwing a `status.UnimplementedError` on all of the cas service implementations that you don't want/need to update yet?  The compiler should do the work of finding the implementations for you.